toolkit/src/components/cut/list-item/action.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ */
+
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+
+export interface DivSignature {
+  Element: HTMLDivElement;
+}
+
+export default class ListItemActionComponent extends Component<DivSignature> {
+  @action
+  onActionClick(event: Event) {
```

Comment:
small tip: since what this function is doing is technically preventing the propagation of the event, maybe it could be called something less generic, more semantic (`onClickStopPropagation` or something similar)